ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_fft.py

```diff
@@ -0,0 +1,34 @@
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# fft
+@handle_frontend_test(
+    fn_tree="jax.numpy.fft.fft",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("float"),
+                                         array_api_dtypes=True,
+                                         shape=helpers.ints(min_value=2).

```

Comment:
I don't see any limitation on the shape of the array mentioned in the documentation. So, I think you shouldn't set the `shape` argument at all.